core/mbed/source/interrupts.cpp

```diff
@@ -26,6 +26,11 @@ void vIRQ_SetVectorX(uint32_t irqn, uint32_t vector, uint32_t flag)
     }
 }
 
+void vIRQ_SetVector(uint32_t irqn, uint32_t vector)
```

Comment:
I still need it as an addressable symbol, though. If I define it as static inline in the header file it should not work, right?